sklearn/feature_extraction/text.py

```diff
@@ -390,9 +413,15 @@ def _validate_custom_analyzer(self):
             pass
 
     def build_analyzer(self):
-        """Return a callable that handles preprocessing, tokenization
-
+        """
+        Return a callable that handles preprocessing, tokenization
```

Comment:
Please revert the added line break.
It should be
```
"""Return ...
```

Reply:
Just updated the format as your suggestions. Thanks!